src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke;
+
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.ToDoTask;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeFormatterBuilder;
+import java.util.ArrayList;
+import java.util.Locale;
+import java.util.Scanner;
+
+
+public class Storage {
+    private final String FILE_PATH;
+    
+    public Storage(String FILE_PATH) {
```

Comment:
Great job at setting `FILE_PATH` as `final`!

However, I am not quite sure if `FILE_PATH` in this case can be considered a constant since you are initializing it in the constructor. Perhaps the naming convention in the case should not be that of a constant.